.gitignore

```
@@ -160,3 +160,4 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+.Rproj.user
```

```suggestion
.Rproj.user
.Rhistory
.RData
```
Sleppa þessum skrám í repoið, og muna að gera 
```bash 
git rm .RData .Rhistory
```
eftir á til að eyða þeim alveg.